scriptoocloud/src/main/resources/config.properties

```diff
@@ -4,3 +4,5 @@ db.username=${db.user}
 db.password=${db.password}
 git.repositories.path = ${git.repositories.path}
 digingUrl = http://diging.asu.edu/index.html
+rootUploadLocation = ${uploadDirectory}
+pageSize = 5
```

Comment:
the default page size should probably be more like 25 or something? This is for datafiles, right?

Reply:
I bumped the `pageSize` up to 25 as requested.  As it stands now, this `pageSize` is for both the `Dataset` list and the `DataFile` list, but if that needs to be adjusted, it's easy enough to do.  I started at 5 just for ease of manual testing.